pybullet build time: Jan  9 2023 19:56:43
pybullet build time: Jan  9 2023 19:56:43
Exception ignored in: <function SAMPLESIMULATION.__del__ at 0x107a53420>
Exception ignored in: <function SAMPLESIMULATION.__del__ at 0x107a53420>
Traceback (most recent call last):
  File "/Users/han/Desktop/mybots/sample_simulation.py", line 30, in __del__
pybullet.error: Not connected to physics server.
Exception ignored in: <function SAMPLESIMULATION.__del__ at 0x107a53420>
Traceback (most recent call last):
  File "/Users/han/Desktop/mybots/sample_simulation.py", line 30, in __del__
pybullet.error: Not connected to physics server.
Exception ignored in: <function SAMPLESIMULATION.__del__ at 0x107a53420>
Traceback (most recent call last):
  File "/Users/han/Desktop/mybots/sample_simulation.py", line 30, in __del__
pybullet.error: Not connected to physics server.
Traceback (most recent call last):
  File "/Users/han/Desktop/mybots/sample_simulation.py", line 30, in __del__
pybullet.error: Not connected to physics server.
Exception ignored in: <function SAMPLESIMULATION.__del__ at 0x107a53420>
Traceback (most recent call last):
  File "/Users/han/Desktop/mybots/sample_simulation.py", line 30, in __del__
pybullet.error: Not connected to physics server.
Exception ignored in: <function SAMPLESIMULATION.__del__ at 0x107a53420>
Traceback (most recent call last):
  File "/Users/han/Desktop/mybots/sample_simulation.py", line 30, in __del__
pybullet.error: Not connected to physics server.
